server/cp/mgmt_api_signals.py

```diff
@@ -2,7 +2,7 @@
 
 
 async def on_company_create(company, **kwargs):
-    company.setdefault("auth_provider", "gip")
+    company.auth_provider = "gip"
```

Comment:
The idea is to provide a default value, here `gip` will **always** be applied. Please change this to assign the `gip` value if no value is applied